content/docs/user-guide/experiment-management/index.md

```diff
@@ -88,7 +96,7 @@ These are the main alternatives:
 - **Git tags and branches** - use the repo's "time dimension" to distribute your
   experiments. This makes the most sense for experiments that build on each
   other. Helpful if the Git [revisions](https://git-scm.com/docs/revisions) can
-  be easily visualized, for example with tools
+  be visualized easily, for example with tools
```

Comment:
minor, style: I would remove "easily"

Reply:
We're not selling DVC here, it's a conditional: if Git revisions can be visualized easily. Easily is key. They can always be visualized (e.g. with `git log` if you know how to use it well), but if it's hard then this doesn't apply.

Reply:
Changed completely to

> Git-based experiment structures are especially helpful along with Git history exploration tools like in GitHub.